src/components/Card/Card.scss

```diff
@@ -0,0 +1,16 @@
+.card {
+    padding: 1rem;
+    display: grid;
+    place-items: center;
+    border-radius: 1rem;
+
+    &__light {
+        background-color: white;
+        box-shadow: 0 0 20px 0 lightgray;
+    }
+
+    &__dark-trans {
```

Comment:
There are 4 requirements for the `card` I have attached the images in the description as well, please double check with that.